src/include/sof/audio/smart_amp_maxim.h

```diff
@@ -0,0 +1,39 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Maxim Integrated All rights reserved.
+ *
+ * Author: Ryan Lee <ryans.lee@maximintegrated.com>
+ */
+
+#ifndef __SOF_AUDIO_SMART_AMP_MAXIM_H__
+#define __SOF_AUDIO_SMART_AMP_MAXIM_H__
+
+#include <sof/platform.h>
+#include <sof/audio/component.h>
+
+#define SMART_AMP_MAX_STREAM_CHAN   8
+
+/** IPC blob types */
+#define SOF_SMART_AMP_CONFIG	0
+#define SOF_SMART_AMP_MODEL     1
+
+struct smart_amp_model_data {
```

Comment:
Can we prefix macros and APIs in this file with MAXIM_ instead of SOF_ as it appears to be MAxim specific.

Reply:
Shall fix the prefix macros and APIs with next rules.
1) generic code : start with smart_amp_, Not Maxim specific but smart amp specific.
2) maxim specific code : start with maxim_
3) sof related part : keep existing name.
   ex) sof_smart_amp_config : same structure is being used in the topology file.